### PR TITLE
fix trade details

### PR DESCRIPTION
**Pull request by sunnya97**
## What is the purpose of the change:

Improve the swap widget
- Better logic around when to show the fiat values
- Better formatting around the fiat values
- Show total output difference in the swap widget
- Reorder elements in the trade details section
- Display 2 decimal points for spread factors in pools

### CODE REVIEW

**Review by fabryscript — APPROVED** [review]
We need to mass translate the changed keys in the `en.json` file.
Everything else seems great for me!

**Comment by sunnya97** [comment]
@fabryscript done

File: packages/web/components/swap-tool/alt.tsx [hunk 1]
```diff
@@ -94,8 +94,18 @@ export const AltSwapTool: FunctionComponent<SwapToolProps> = observer(
...
 
+    const outputDifference = new RatePretty(
+      swapState.inAmountInput?.fiatValue
+        ?.toDec()
+        .sub(swapState.tokenOutFiatValue?.toDec())
+        .quo(swapState.inAmountInput?.fiatValue?.toDec()) ?? new Dec(0)
+    );
+
+    const showOutputDifferenceWarning =
+      outputDifference.toDec().abs().gt(new Dec(0.05)) ?? false;
```
Comment (crnbarr93) [inline packages/web/components/swap-tool/alt.tsx:105]: nit: is this coalescing operator necessary?

File: packages/web/components/swap-tool/alt.tsx [hunk 2]
```diff
@@ -484,18 +502,35 @@ export const AltSwapTool: FunctionComponent<SwapToolProps> = observer(
...
+                          {formatPretty(swapState.tokenOutFiatValue, {
+                            ...getPriceExtendedFormatOptions(
+                              swapState.tokenOutFiatValue.toDec()
+                            ),
+                          })}
+                          <span
+                            className={
+                              showOutputDifferenceWarning
+                                ? "text-rust-400"
+                                : "text-osmoverse-700"
```
Comment (sunnya97) [inline packages/web/components/swap-tool/alt.tsx:526]:
```suggestion
                                : "text-osmoverse-600"
```

**Review by crnbarr93 — APPROVED** [review]

File: packages/web/components/swap-tool/alt.tsx (hunk 1 above)
Comment (sunnya97) [reply packages/web/components/swap-tool/alt.tsx:105]:
```suggestion
      outputDifference.toDec().abs().gt(new Dec(0.05));
```

**Pull request merged**